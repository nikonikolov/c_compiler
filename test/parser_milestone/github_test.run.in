
int x()
{}

int g, b, d, f=a;
int zz(int a, int b, int c)
{
  if(a==b)
    a=b;
  else{
    int fsdfsdfs;
  }

	while(a==b){
		a=b;
	}

	for(i=a; i<a; i=b){
		a=b;
	}  
}