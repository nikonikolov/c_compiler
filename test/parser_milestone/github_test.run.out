VARIABLE : g
VARIABLE : b
VARIABLE : d
VARIABLE : f
FUNCTION : x
FUNCTION : zz
    PARAM : a
    PARAM : b
    PARAM : c
SCOPE
    SCOPE
        VARIABLE : abcd
        (abcd)=((a)+(b))
    (a)=(b)
    SCOPE
        VARIABLE : fsdfsdfs
        RETURN (a)==(b)
    SCOPE
        (a)=(b)
    SCOPE
        (a)=(b)
    RETURN (a)==(b)
